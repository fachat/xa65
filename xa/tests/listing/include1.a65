
	.org $1000

#include "include1a.a65"

.include "include1a.a65"


