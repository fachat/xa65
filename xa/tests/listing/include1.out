
include1.a65


    2 A:1000                                     *= $1000

include1a.a65


    2 A:1000  a9 00                              lda #$00

include1.a65


    6 A:1002                                     .include "include1a.a65"

include1a.a65


    2 A:1002  a9 00                              lda #$00

include1.a65

